content/developer/howtos/localization.rst

```diff
@@ -0,0 +1,370 @@
+
+.. note::
+
+  Recommended **xmlid** for the record is `chart_template`.
+  If you need many chart of accounts, you can add some suffixes, i.e. `chart_template_XXX`.
+
+
+Chart of Accounts
+=================
+
+Accounts
+--------
+
+.. seealso::
+   - :ref:`Account References <reference/account_account>`
+   - :doc:`/applications/finance/accounting/payables`
+   - :doc:`/applications/finance/accounting/receivables`
+
+Obviously, *Chart of Accounts* cannot exist without *Accounts*. You need to specify them in `data/account.account.template.csv`.
+
+.. example::
+  `addons/l10n_ch/data/account.account.template.csv <https://github.com/odoo/odoo/blob/15.0/addons/l10n_ch/data/account.account.template.csv>`__.
+
+  .. literalinclude:: {ODOO_RELPATH}/addons/l10n_ch/data/account.account.template.csv
+    :language: csv
+    :end-at: ch_coa_1171
+
+XML is prefered but you may use xml format instead.
+
+.. example::
+  `addons/l10n_at/data/account_account_template.xml <https://github.com/odoo/odoo/blob/15.0/addons/l10n_at/data/account_account_template.xml>`__.
+
+  .. literalinclude:: {ODOO_RELPATH}/addons/l10n_at/data/account_account_template.xml
+    :language: xml
+    :start-at: chart_at_template_0010
+    :end-at: </record>
+
+.. warning::
+
+    Avoid the usage of liquidity ``account.account.type``!
+    Indeed, the bank & cash accounts are created directly at the installation of the localization module and then, are linked to an ``account.journal``.
+
+.. warning::
+
+    Only one account of type payable/receivable is enough.
+
+.. warning::
+
+    Don't create too much accounts: 200-300 is enough.
+
+Next settings for the chart of accounts are set in a separate file, because we need to provide `list of accounts <#accounts>`__ first. In `data/account_chart_post_data.xml` we set some default accounts:
+
+.. todo add reference to account_id in CoA
+
+.. example::
+  `addons/l10n_ch/data/l10n_ch_chart_post_data.xml <https://github.com/odoo/odoo/blob/15.0/addons/l10n_ch/data/l10n_ch_chart_post_data.xml>`__.
+
+  .. literalinclude:: {ODOO_RELPATH}/addons/l10n_ch/data/l10n_ch_chart_post_data.xml
+    :language: xml
+    :start-at: l10nch_chart_template
+    :end-at: </record>
+
+Account tags
+------------
+
+.. seealso::
+   :ref:`Account Tag References <reference/account_account_tag>`
+
+Tags is a way to sort accounts.
+For example, imagine you want to create a financial report having multiple lines but you have no way to find a rule to dispatch the accounts according their ``code`` or ``name``.
+The solution is the usage of tags, one for each report line, to filter accounts like you want.
+
+Put the tags in `data/account_account_tag_data.xml` file:
+
+Then apply tags on some accounts (field `tag_ids`).
+
+.. example::
+  `addons/l10n_lt/data/account.account.template.csv <https://github.com/odoo/odoo/blob/15.0/addons/l10n_lt/data/account.account.template.csv>`__.
+
+  .. literalinclude:: {ODOO_RELPATH}/addons/l10n_lt/data/account.account.template.csv
```

Comment:
would probably present better with a `csv-table + file` directive. Can you use `start at`/`end-at` as well in that?